e2e/tests/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -0,0 +1,149 @@
+/*****************************************************************************
```

Comment:
Do we want to remove this one for the time being, since this will be added in the Restricted Notebook PR?